packages/app/src/specs/SpecsList.cy.tsx

```diff
@@ -42,21 +41,24 @@ describe('<SpecsList />', { keystrokeDelay: 0 }, () => {
         acc.relative.length < spec.relative.length ? spec : acc
       , specs[0])
 
-      cy.get(inputSelector).type('garbage 🗑', { delay: 0 })
+      cy.findByLabelText(defaultMessages.specPage.searchPlaceholder)
+      .as('specsListInput')
+
+      cy.get('@specsListInput').type('garbage 🗑', { delay: 0 })
```

Comment:
Not your change but what does `delay: 0` do here? (why do we need it)

Reply:
I think it's just cumulatively saving some time across tests (vs the 10ms default)? We do it several places, @JessicaSachs might know if there's a bigger reason.